src/backend/marsha/core/api/video.py

```diff
@@ -723,6 +737,100 @@ def push_attendance(self, request, pk=None):
             status=404,
         )
 
+    @action(detail=False, methods=["post"])
+    # pylint: disable=unused-argument
+    def init_display_username(self, request, pk=None):
+        """View handling getting display_username. Create or get registration."""
```

Comment:
I'm not comfortable with this endpoint url and the description. Getting something making a `POST` request is a little bit strange. 
The enpoint could be `/api/liveregistrations/display_username` (or `display_name` but it's linked to the implementation on the other PR). The `POST` create it and the `PATCH` update it.

Reply:
And IMO if the resource already exists, it should not be updated. You should return something else than a `200` status code and the front application is responsible to act according to this response status. After few research returning a `409` seems correct.

Reply:
After some research and reflexion, merging the `POST` and `PATCH` endpoints in one and using the `PUT` verb can be a solution. This reponse on stackoverflow can match our need https://stackoverflow.com/a/630475